tests/integration/components/paper-form-field-test.js

```diff
@@ -198,4 +198,43 @@ module('Integration | Component | {{paper-form-field}}', function(hooks) {
     assert.equal(this.get('object.name'), 'John White',
                  `doesn't update object value`);
   });
+
+  test(`Shows a after content depending on the given content`,
+  async function(assert) {
+    this.set('object', EmberObject.create({
+      text: 'example'
+    }));
+
+    await render(hbs`
+      {{paper-form-field type="text-field"
+                         object=object
+                         after="after text"
+                         propertyPath="text"
+                         updated=(action 'update')}}`);
+
+    let $input = this.element.querySelector('input.md-input').value;
+    assert.equal($input, 'example', 'shown component has correct value');
```

Comment:
You don't have to check the value in this test, that is already tested in another test.